### ISSUE TITLE
Doc search failing because of undefined jQuery reference

### ISSUE THREAD

**Issue by neilclemons** [mozman/ezdxf#937]
When loading the documentation and attempting to search, the Searching... indicator hangs indefinitely. Console reports two undefined references to jQuery in search.html, one of which is supposed to load the search index script.

I can use a local override with the following code in searchtools.js and search results will load, but the console shows a bunch of 404 errors and every search result seems to return a 403 when clicking through.

```
var jq = document.createElement('script');
jq.onload = function() {
  jQuery(function() {
    Search.loadIndex("searchindex.js");
  });
}
jq.src = "https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js";
document.getElementsByTagName('head')[0].appendChild(jq);
```

It looks like Sphinx removed the jQuery requirement some time ago. Perhaps the docs need to be regenerated with an updated version?

**Comment by mozman** [comment]
That's right. I use `Sphinx` as a black box, if it doesn't work - it doesn't work. 

The version i used was 7.2.5 and is now updated to 7.2.6 but does not solve the issue. I also updated the `sphinx-rtd-theme` to 2.0.0rc2 because someone solved a similar issue in 2020 by that - but not today.

The search is broken on my site https://ezdxf.mozman.at/docs/ and on https://ezdxf.readthedocs.org. I don't have a solution for this and Google doesn't seem to have one either.

Use Google to search the docs like `ezdxf line entity`, it's maybe faster and better. 😄
